example/profile.php

```diff
@@ -25,22 +35,46 @@
 <html>
    <head>
         <meta charset="utf-8">
-        <title>PROFILE PAGE</title>
+        <title>YOTI PROFILE</title>
```

Comment:
No uppercase

Reply:
That's fine I can use snake-case format.